in/cities/aalborg.stat.xml

```diff
@@ -1,4 +1,4 @@
 <city>
-    <general inhabitants="140000" households="66666"/>
+    <general inhabitants="117351" households="84453"/>
```

Comment:
```suggestion
    <general inhabitants="117351" households="55881"/>
```

Reply:
I am not sure how you got 84453, if you divide inhabitants by 2.1 as with other cities, you get 55881

Reply:
Huh, weird - you're right. Checked the others and they're correct.